### DISCUSSION TITLE
Welcome to liquidprompt Discussions!

### DISCUSSION THREAD

**Discussion by nojhan** [liquidprompt/liquidprompt#683] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Rycieos** [comment]
Hello! I am the lead maintainer of Liquidprompt. I'll be the person primarily interacting on Discussions. I hope to hear lots of feedback from the community here; what you like about Liquidprompt, what frustrates you, or even meta things like changes to our GitHub organization. Just remember to keep things nice!

I look forward to hearing from you!

**Comment by augmentedfourth** [comment]
Hi! I've contributed a little bit of code to Liqudprompt over the years, but these days I mostly just manage the distribution/packaging for the tool's formula on Homebrew/Linuxbrew.
